File: 2. hét/Szülő-gyerek/SzuloGyerek.cpp
#include <iostream>

class Madar {};

class Pingvin : public Madar
{
	public:
		virtual void nemrepul() {std::cout<<"Nem repül.";}
};

int main (int argc, char **argv)
{
	Madar *madar=new Pingvin();
	//meghívja a szülő sétál metódusát
	madar->nemrepul();
}

File: 2. hét/Szülő-gyerek/SzuloGyerek.java
class Madar
{}

class Pingvin extends Madar
{
	public void nemrepul()
		{
		System.out.println("Nem repül.");
		}
}

public class SzuloGyerek
{
	public static void main(String args[])
	{
		Madar madar=new Pingvin();
		madar.nemrepul();

	}
}
